services/QuillLMS/app/workers/google_student_importer_worker.rb

```diff
@@ -8,13 +8,11 @@ class GoogleStudentImporterWorker
 
   def perform(teacher_id, context = "none", selected_classroom_ids=nil)
     teacher = User.find(teacher_id)
+    return unless teacher.google_authorized?
```

Comment:
Do we know how this state happens: a google import job for the teacher is fired, but the teacher isn't `google_authorized`?

Wondering because:
1. Is there is a bug further upstream where this job shouldn't fire?
2. Should we send a `Pusher` notification for that non-authorized state (we send one for the success state)? If it's actionable and they will be on a page to receive the message, that might be helpful information.

Reply:
1)  That's a good call.  I have it checked for authorization [here](https://github.com/empirical-org/Empirical-Core/blob/develop/services/QuillLMS/app/controllers/auth/google_controller.rb#L48) but I missed the check [here](https://github.com/empirical-org/Empirical-Core/blob/develop/services/QuillLMS/app/controllers/teachers/classroom_manager_controller.rb#L217).  I will fix that.
2) The pusher notification is a great suggestion and a good fallback until we implement the reauthorization UI like we have for clever.  I will add that in.

Reply:
@dandrabik Re-requesting review due to the new changes.  I've also added a screenshot showing the new pusher notification.